plugins/pug.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pug
+spec:
+  shortDescription: >-    
+    Checks for deprecated objects in a cluster 
+  homepage: https://github.com/rikatz/kubepug
+  caveats: |
+    * Kubepug downloads Kubernetes swagger.json to check objects and which are
```

Comment:
I just read this as "the swagger.json is deprecated". Probably not what you mean :)

Maybe change to something like

> By default, pug finds deprecated object relative to the current kubernetes master branch. To target a different kubernetes release, use the --k8s-version argument.